docs_versioned_docs/version-ros2humble/ros/installation/controller.mdx

```diff
@@ -54,8 +55,10 @@ agent on
 scan on
 ```
 
-4. Put the controller in pairing mode. Press and hold the PS and Share buttons on your controller until the LED begins rapidly flashing white.
-5. The bluetooth scan will display the MAC addresses of nearby devices. Determine which MAC address corresponds to the controller and copy it. In the bluetooth control application, run:
+4. Put the controller in pairing mode. Press and hold the PS and Share buttons on your controller until
+the LED begins rapidly flashing white.
```

Comment:
consider adding spaces to align the text on separate lines (assuming this was an intentional carriage return)